test/src/logging.rs

```diff
@@ -28,14 +28,16 @@ use tracing_subscriber::{EnvFilter, FmtSubscriber};
 pub fn init() {
     if env_logger::builder().is_test(true).try_init().is_ok() {
         if env::var("RUST_LOG").is_err() {
-            env::set_var("RUST_LOG", "error");
+            env::set_var("RUST_LOG", "debug");
         }
 
-        let mut builder = FmtSubscriber::builder().with_env_filter(EnvFilter::from_default_env());
+        let mut builder = FmtSubscriber::builder()
+            .with_env_filter(EnvFilter::from_default_env())
+            .with_test_writer();
```

Comment:
Wait, is that the new thing? `env_logger::builder().is_test(true)` did previously have the same effect, so what's the interaction between the two?

Reply:
`.is_test(true)` on `env_logger()` has no effect on the tracing subscriber. I’m still seeing error logs for test cases that pass by default. And if I change the default log level I see logs for every test case.

Reply:
Oh interesting, I always assumed this would log _though_ `env_logger`. So this never worked as intended? Is the `.is_test(true)` needed at all?

Reply:
I think creating an `env_logger` is not needed at all. If the [`tracing-log`](https://docs.rs/tracing-subscriber/0.2.19/tracing_subscriber/index.html#optional-dependencies) feature on `tracing_subscriber` is enabled (which it is by default), the `fmt` subscriber will print all events emitted by `log` according to the docs.